.github/workflows/main.yml

```diff
@@ -316,3 +316,157 @@ jobs:
         name: Realm.UnityWeaver.${{ steps.find-nupkg-version.outputs.package_version }}.nupkg
         path: ${{ github.workspace }}/Realm/packages/Realm.UnityWeaver.${{ steps.find-nupkg-version.outputs.package_version }}.nupkg
         retention-days: ${{ github.event_name != 'pull_request' && 30 || 1 }}
+  run-tests-net-framework:
+    runs-on: windows-latest
+    name: Run tests for NET framework
+    needs: build-packages
+    steps:
+    - name: Add msbuild to PATH
+      uses: microsoft/setup-msbuild@v1.0.2
+    - name: Checkout code
+      uses: actions/checkout@v2
```

Comment:
The versioning here (`checkout@v2`, `download-artifact@v2`) appears in multiple locations.
Is this likely to change at some point and if so would it make sense to have the version extracted to one common location?

Reply:
This is the generated workflow file - the template will only specify the versions once: https://github.com/realm/realm-dotnet/blob/6a036617f714bb969bf938588b9fa555abf1adab/.github/templates/main.yml#L11-L16

If we wanted to change it, this is the only place that would need adjustment.

Reply:
Alright, sounds great. 🚀